examples/CIB/ex1/main.cpp

```diff
@@ -399,6 +495,83 @@ main(int argc, char* argv[])
             pout << "+++++++++++++++++++++++++++++++++++++++++++++++++++\n";
             pout << "\n";
 
+            // Fill ghost cells of pressure and velocity to compute hydrodynamic forces.
```

Comment:
Makes sense. We intended to have this code be "user facing": depending on the application, the user can choose where and around which objects to place control volumes. Makes sense to automate ghost cell filling.